package.json

```diff
@@ -89,7 +89,8 @@
 		"valid-url": "^1.0.9"
 	},
 	"_moduleAliases": {
-		"node_helper": "js/node_helper.js"
+		"node_helper": "js/node_helper.js",
+		"logger": "js/logger.js"
```

Comment:
Good thinking, like the idea. Too bad node_helper is already widely used, otherwise I'd propose to add some prefix to those aliases so that develoeprs would know that they are from the MM-core.